src/keys/utils.ts

```diff
@@ -41,18 +47,71 @@ async function generateKeyPair(bits: number): Promise<KeyPair> {
   return await generateKeyPair({ bits });
 }
 
+function generateMnemonic(words: 12 | 24 = 24): string {
+  switch (words) {
+    case 12:
+      return Bip39.generateMnemonic(128);
+    case 24:
+    default:
+      return Bip39.generateMnemonic(256);
+  }
+}
+
+function generateSeedFromMnemonic(mnemonic: string) {
+  if (!isValidMnemonic(mnemonic)) throw new keysErrors.ErrorMnemonicInvalid();
+  return Bip39.mnemonicToEntropy(mnemonic);
+}
+
+function isValidMnemonic(mnemonic: string): boolean {
+  if (mnemonic.split(' ').length !== 24) return false;
```

Comment:
This is incorrect. We allow 12 or 24 words. Besides this is also unnecessary, just rely on `validateMnemonic`.

We should change the whole function to be:

```
validateRecoveryCode
```

This is different from a `verify` prefix which would be checking if it is correct to our keynode. Validate is for checking format/syntax.

Reply:
This is now `validateRecoveryCode`.